### PR TITLE
Implement AdMob/AppLovin native ad

### PR DESCRIPTION
**Pull request by github-lucas-bon**
Sample

https://user-images.githubusercontent.com/99195905/164482538-e9547c4a-143c-450e-bcee-21e04555241c.mp4

### CODE REVIEW

**Review by github-benjamin-volland — APPROVED** [review]

File: TeadsSDKDemo/app/src/main/java/tv/teads/teadssdkdemo/format/mediation/adapter/AppLovinNativeRecyclerViewAdapter.kt [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    }
+
+    private fun createNativeAdView(): MaxNativeAdView {
+        val binder: MaxNativeAdViewBinder = MaxNativeAdViewBinder
+            .Builder(if (isGrid) R.layout.item_applovin_native_ad_grid else R.layout.item_applovin_native_ad)
+            .setTitleTextViewId(R.id.ad_title)
+            .setBodyTextViewId(R.id.ad_body)
+            .setMediaContentViewGroupId(R.id.media_view_container)
+            .setOptionsContentViewGroupId(R.id.ad_options_view)
+            .setIconImageViewId(R.id.dummy_icon_image_view) // todo adapter sdk should be fixed
```
Comment (github-lucas-bon) [inline TeadsSDKDemo/app/src/main/java/tv/teads/teadssdkdemo/format/mediation/adapter/AppLovinNativeRecyclerViewAdapter.kt:124]: Waiting new version from android sdk to remove it

File: TeadsSDKDemo/app/src/main/res/layout/item_applovin_native_ad.xml [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+            android:background="@drawable/custom_button_selector"
+            android:gravity="center"
+            android:text="more"
+            android:textColor="@color/textColorNoBg"
+            android:textSize="12sp"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/ad_body" />
+
+        <!-- todo it was added because teads adapter is obligating me to add it, but it should not -->
```
Comment (github-lucas-bon) [inline TeadsSDKDemo/app/src/main/res/layout/item_applovin_native_ad.xml:110]: Waiting new version from android sdk to remove it

File: TeadsSDKDemo/app/src/main/res/layout/item_applovin_native_ad_grid.xml [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+            android:background="@drawable/custom_button_selector"
+            android:gravity="center"
+            android:text="more"
+            android:textColor="@color/textColorNoBg"
+            android:textSize="12sp"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintStart_toStartOf="@id/sponsor_label"
+            app:layout_constraintTop_toBottomOf="@id/sponsor_label" />
+
+        <!-- todo it was added because teads adapter is obligating me to add it, but it should not -->
```
Comment (github-lucas-bon) [inline TeadsSDKDemo/app/src/main/res/layout/item_applovin_native_ad_grid.xml:107]: Waiting new version from android sdk to remove it

File: TeadsSDKDemo/build.gradle [hunk 4]
```diff
@@ -15,6 +15,7 @@ allprojects {
         google()
         jcenter()
         maven {url "https://teads.jfrog.io/artifactory/SDKAndroid-maven-prod"}
+        maven {url "https://teads.jfrog.io/artifactory/SDKAndroid-maven-earlyAccess"} // todo temp
```
Comment (github-lucas-bon) [inline TeadsSDKDemo/build.gradle:18]: Remove it

**Pull request merged**